lib/iris/fileformats/nimrod_load_rules.py

```diff
@@ -25,58 +28,306 @@
 NIMROD_DEFAULT = -32767.0
 
 TIME_UNIT = cf_units.Unit(
-    "hours since 1970-01-01 00:00:00", calendar=cf_units.CALENDAR_GREGORIAN
+    "seconds since 1970-01-01 00:00:00", calendar=cf_units.CALENDAR_GREGORIAN
 )
 
 
+DEFAULT_UNITS = {
+    817: "m s^-1",
+    804: "knots",
+    422: "min^-1",
+    218: "cm",
+    172: "oktas",
+    155: "m",
+    101: "m",
+    63: "mm hr^-1",
+    61: "mm",
+    58: "Celsius",
+    12: "mb",
+    8: "1",
+}
 FIELD_CODES = {73: "orography"}
-VERTICAL_CODES = {0: "height", 1: "altitude", 12: "levels_below_ground"}
+# VERTICAL_CODES contains conversions from the Nimrod Documentation for the
+# header entry 20 for the vertical coordinate type
+VERTICAL_CODES = {
```

Comment:
Use of this is private to 'vertical_coord'.
Though in this case, at least the name gives clear scope.

Reply:
Moved into the relevant functions.